profile/CSIP.xml

```diff
@@ -1026,10 +1026,10 @@
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP86" REQLEVEL="MUST" EXAMPLES="structMapExample1 structMapExample2">
+            <requirement ID="CSIP86" REQLEVEL="MUST" RELATEDMAT="CSIP1" EXAMPLES="structMapExample1 structMapExample2">
                 <description>
                     <head>Main structural division label</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The main division (div) element in the package uses the package ID as the value for the attribute LABEL.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The main division (div) element in the package uses the element named "Content Identification" (CSIP1, mets/@OBJID) as the value for the attribute LABEL.</p>
```

Comment:
The package MET document's main division (div) element has an @LABEL attribute value identical to the root mets element's @OBJID attribute.

Reply:
The package's top-level structural division (div) element must have an @Label attribute value identical to the package identifier, i.e. the same value as the mets/@OBJID attribute.